src/utils/chat/promptGenerator.ts

```
@@ -486,16 +486,45 @@ ${
+    if (!customInstruction && !currentAssistant) {
       return null
     }
-    return {
-      role: 'user',
-      content: `Here are additional instructions to follow in your responses when relevant. There's no need to explicitly acknowledge them:
+    
+    // Build prompt content
+    let content = ''
+    
+    // Add assistant's system prompt (if available)
+    if (currentAssistant && currentAssistant.systemPrompt) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)